build/template-OneBranch-CI-libsandsamples.yaml

```diff
@@ -6,6 +6,7 @@ parameters:
   BuildConfiguration: 'release'
   MsalClientSemVer: '4.60.0-devopsbuild' 
   MsalSourceDir: 'microsoft-authentication-library-for-dotnet\' #Default MSAL Location
+  Mode: 'build' # Default to 'build'
```

Comment:
When is this set to "test" and when it is set to "build"?  Is it related to OneBranch? I think we already have env var for OB?

Reply:
See `$(PipelineType)`

Reply:
OB pipelines tasks was calling into classic YAML [here](build/template-OneBranch-Tests-libsandsamples.yaml). So when OB tests run we were still referencing to nuget.org

The fix is to point OB tests to use OB CI yaml. But there are two paths one for build and one for tests. So tweaking paths based on build runs or test runs.